py/release.py

```diff
@@ -170,9 +179,12 @@ def build_operator_image(root_dir, registry, project=None, should_push=True):
 
   image_base = registry + "/tf_operator"
 
-  n = datetime.datetime.now()
-  image = (image_base + ":" + n.strftime("v%Y%m%d") + "-" +
-           commit)
+  if not version_tag:
+    logging.info("No version tag specified; computing tag automatically.")
+    n = datetime.datetime.now()
+    version_tag = n.strftime("v%Y%m%d") + "-" + commit
+  logging.info("Using version tag: %s", version_tag)
+  image = image_base + ":" + version_tag
```

Comment:
are we using python 3.6 yet?

Reply:
No; although I suspect updating our test infra to be 3.6 compatible shouldn't be too bad.